text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,502 @@
+---
+rfc pr: [#347](https://github.com/aws/aws-cdk-rfcs/pull/347)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/66
+---
+
+# CDK Self Managed Stack Set Support
+As a CDK user, I would like to define stack set infrastructure through native CDK constructs and create/update self-managed stack sets through CDK deploy command.
+
+* **Original Author(s):** @linsona
+* **Tracking Issue**: #66
+* **API Bar Raiser**: @skinny85
+
+## Working Backwards
+
+### CHANGELOG
+- feat(core): New Stack Set core type 
+- feat(assembly): New stack set artifcat type and schema
+- feat(cli): ```cdk deploy``` collects and creates/updates stack set 
+- feat(lib): API Calls/Logic for create/update stack sets
```

Comment:
This is not really a customer-facing feature, so we wouldn't tag it with `feat` in the Changelog 😉. Let's remove it from the list.